plaso/cli/storage_media_tool.py

```diff
@@ -684,6 +777,48 @@ def _PromptUserForAPFSVolumeIdentifiers(
 
     return selected_volumes
 
+  def _PromptUserForLVMVolumeIdentifiers(
+      self, volume_system, volume_identifiers):
+    """Prompts the user to provide LVM volume identifiers.
+
+    Args:
+      volume_system (dfvfs.LVMVolumeSystem): volume system.
+      volume_identifiers (list[str]): volume identifiers including prefix.
+
+    Returns:
+      list[str]: selected volume identifiers including prefix or None.
+    """
+    print_header = True
+    while True:
+      if print_header:
```

Comment:
Why not just print the header outside the "while" loop?
print_header is not set to true ever again

Reply:
good question, print_header was mainly used because to keep it consistent with other similar sections in this file, but there seems no need for it, I've removed it in the other places as well.